json/src/spec/account.rs

```diff
@@ -99,6 +99,15 @@ mod tests {
 		assert!(!deserialized.is_empty());
 	}
 
+	#[test]
+	#[should_panic]
+	fn account_balance_panics_over_u256_limit() {
```

Comment:
I don't understand the point of this test? Can you explain? 

I think `uint_deserialization_error_for_hex_too_large` is sufficient for this